services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -413,20 +413,12 @@ const mapStateToProps = (state) => ({
   userRoles: getUserAccessData(state),
 });
 
-const mapDispatchToProps = (dispatch) =>
-  bindActionCreators(
-    {
-      change,
-    },
-    dispatch
-  );
-
 export default compose(
-  connect(mapStateToProps, mapDispatchToProps),
+  connect(mapStateToProps, null),
   reduxForm({
-    form: FORM.EXPLOSIVES_PERMIT,
+    form: FORM.EXPLOSIVES_PERMIT_NEW,
     touchOnBlur: true,
     validate: validateBusinessRules,
```

Comment:
is this defined? I don't see it anywhere

Reply:
Yeah, it's within this same form at line 42

Reply:
Oh cool.  Didn't mess with that part at all.  I'll get rid of it.

Reply:
I am also not keen on this file removing type definitions and prop defaults. I just don't know why this looks like an even older version of the form than the file it's replacing so I'd probably want to do a diff with the other version for other changes.